spec/controllers/static_pages_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe StaticPagesController, type: :controller do
+
+  describe 'get #home' do
+    before { get :home }
+    it 'renders the home view' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.